awx/ui_next/src/screens/Application/ApplicationDetails/ApplicationDetails.jsx

```diff
@@ -21,11 +21,11 @@ function ApplicationDetails({
   const history = useHistory();
   const {
     isLoading: deleteLoading,
-    deletionError,
-    deleteItems: handleDeleteApplications,
+    error: deletionError,
+    request: deleteApplications,
     clearDeletionError,
-  } = useDeleteItems(
-    useCallback(async () => {
+  } = useCallback(
+    useRequest(async () => {
```

Comment:
This should be reversed to `useRequest(useCallback(...`

Reply:
@keithjgrant, update per comments.